test-helpers/src/connection/kafka/mod.rs

```diff
@@ -164,6 +165,28 @@ impl KafkaConsumer {
             }
         }
     }
+
+    /// The offset to be committed should be lastProcessedMessageOffset + 1.
+    pub fn assert_commit_offsets(&self, offsets: HashMap<TopicPartition, i64>) {
+        match self {
+            #[cfg(feature = "kafka-cpp-driver-tests")]
+            Self::Cpp(cpp) => cpp.commit(&offsets, rdkafka::consumer::CommitMode::Sync),
```

Comment:
The KafkaConsumerCpp and KafkaConsumerJava should be implementing a common interface.
So they should both have the same method names and take the same arguments.